projeto-gerenciador-gastos/src/View/View.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace GGP\View;
+
+class View
+{
+    /**
+     * Caminho para URL
```

Comment:
Acredito que aqui não queria ser referir a url mas ao path da view correto? Se for isso mesmo ajustar o comentário por favor